app/src/main/AndroidManifest.xml

```diff
@@ -155,7 +152,12 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.walkthrough.WalkthroughActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar" >
```

Comment:
This change should not be added in the PR as this is for testing purpose only.

Reply:
Sure, I should mark the PR as draft for now

Reply:
Done